terraform/aws/components/gpc-consumer/locals.tf

```diff
@@ -0,0 +1,23 @@
+locals {
+  default_tags = {
+    Project = var.project
+    Environment = var.environment
+    Component = var.component
+  }
+
+  resource_prefix = "${var.project}-${var.environment}-${var.component}"
+
+  availability_zones = ["${var.region}a", "${var.region}b", "${var.region}c"]
+
+  
+  # Use below when the ECR repo is created by terraform in account component.
+  #image_name = "${data.terraform_remote_state.account.outputs.nhais_ecr_repo_url}:${var.nhais_build_id}"
+  image_name = "${var.account_id}.dkr.ecr.${var.region}.amazonaws.com/gpc-consumer:${var.gpc-consumer_build_id}"
+  wiremock_image_name = "${var.account_id}.dkr.ecr.${var.region}.amazonaws.com/gpc-consumer-wiremock:${var.gpc-consumer_build_id}"
```

Comment:
is wiremock needed for gpc?

Reply:
Matt mentioned it in https://gpitbjss.atlassian.net/browse/NIAD-1154